File: backend-springboot/src/main/java/com/javeriana/pontimovil/ponti_movil/dto/gestion_rutas/estacion/rEstacionDTO.java
package com.javeriana.pontimovil.ponti_movil.dto.gestion_rutas.estacion;

import lombok.*;
import java.io.Serializable;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@EqualsAndHashCode
@NoArgsConstructor
public class rEstacionDTO implements Serializable {
    UUID id;
    String nombre;
    Integer orden;
    rDireccionDTO direccion;
    Boolean dentroRuta;
}

File: frontend-angular/src/app/dto/gestionar-rutas/estacion/r-estacion-dto.ts
import {RDireccionDTO} from './r-direccion-dto';

export class REstacionDto {
  constructor(
    public id: number,
    public nombre: string,
    public  orden: string,
    public direccion: RDireccionDTO,
    public dentroRuta: boolean
  ) {}
}
